desktop-activity.md

```diff
@@ -0,0 +1,200 @@
+in the `actvity.info` file.
+
+Note that the Sugar Activity icon must follow the guidelines as
+decribed in [The Sugar Interface:
+Icons](https://wiki.sugarlabs.org/go/Human_Interface_Guidelines/The_Sugar_Interface/Icons).
+
+There is a helper script, [Sugar
+Iconify](https://wiki.sugarlabs.org/go/Sugar_iconify) that will help
+you create Sugar-compliant icons.
+
+Of course, the interesting changes will be the ones you make to the
+activity itself. Below you will find some links to some resources
+regarding some Activity devolepment details, but perhaps the best way
+to get started is to modify an existing activity.
+
+### Running your new activity
+
+Launch Sugar and your new activity should be immediately available,
+although since it has not been selected as a **favorite**, it will not
+appear by default on the Sugar Home View. You need to select the
+**List View** to see your activity's icon.
+
+Click on the icon and, if all goes well, your activity will run!!!
+
+Of course, there are many opportunities to make mistakes. Don't get
+discouraged, as engaging in debugging is a great way to learn. One
+useful tool is the Log Activity, which will show you the log files of
+the system and individual activities. Alternatively, you can look at
+the log files from the command line.
+
+    ~/sugar-build/home/dotsugar/default/logs
+
+### File structure
+
+In your new activity, you will find the following file structure:
+
+    my-activity/
+    |-- activity/
+    |   |-- activity.info
+    |   `-- activity-icon.svg
+    |-- activity.py
+    `-- setup.py
+
+* `activity/` contains information about your activity, including the
+  name, ID, and the icon.
+
+* `activity.py` contains an instance of the activity class, which is
+  run when your activity is launched.
+
+* `setup.py` lets you install your activity or make an installable
+  bundle with it
+
+Now you are ready to go ahead and develop your activity in the html,
+js and css files.
+
+### Translation
+
+Since Sugar serves a global audience, it is important to enable your
+activity for internationalization and localization. A [guide to best
+practices](https://wiki.sugarlabs.org/go/Translation_Team/i18n_Best_Practices)
+is found in our wiki.
+
+### Revision control your code
+
+For development you can initialize the repository as a git
+repository. This will help you to track your changes. First use [git
+init](https://www.kernel.org/pub/software/scm/git/docs/git-init.html)
+to initialize the repository:
+
+    git init
+
+With [git
+status](https://www.kernel.org/pub/software/scm/git/docs/git-status.html)
+you can show the available files in the folder they are still
+untracked. Now add all the files in the directory besides the lib
+folder and commit those changes, you can use git status again to see
+the current state:
+
+    git add .
+    git commit -a -m 'Initial import'
+    git status
+
+We recommend that you use [github](http://github.com) to host your
+Sugar Activity.
```

Comment:
Sugar Activity, and we welcome you to transfer your activity to the [github.com/sugarlabs](https://github.com/sugarlabs) github organization so that it is easier for the Sugar community to find and participate in development.